### ISSUE TITLE
Titles and Descriptions not saving with Gutenberg

### ISSUE THREAD

**Issue by iliyasdxb** [semperfiwebdesign/all-in-one-seo-pack#2045]
Since updating All In One SEO Pack Pro and using Gutenberg editor the site it longer picks up the correct SEO Title or Description. Even when I manually edit these they no longer save. If I switch back to the Classic Editor it is ok and works as normal. Up until we recently updated All In One SEO Pack Pro and renewing our subscription it was working fine.
Is there a bug or are we doing something wrong?

**Comment by michaeltorbert** [comment]
@iliyasdxb You can submit your login information to our secure form here and we'll take a look.

**Comment by iliyasdxb** [comment]
where is the form?

**Comment by michaeltorbert** [comment]
Oops. Sorry I was sure I'd pasted it. ;)
https://semperplugins.com/contact/support-login-information/

**Comment by iliyasdxb** [comment]
OK, I will set up a user for you now and send it

**Comment by michaeltorbert** [comment]
I've verified the issue. For some reason, when creating a new post, data already exists in the AIOSEOP title and description fields (and social title/description). When changing it and saving, it doesn't save correctly.
I'm going to delete extraneous comments from the thread for brevity's sake.
@iliyasdxb I'm cloning the site to a private development server so we can fully investigate and develop a fix without messing around on your live site.

**Comment by iliyasdxb** [comment]
Thanks.

Best Regards,
Iliyas Campbell
iliyasurg@me.com
07379 815 663

**Comment by wpsmort** [comment]
I've identified this as a bug with the SiteOrigin Widgets Bundle plugin.  It runs a query using update_meta_chase() against the wp_postmeta table against the wrong post ID.  This causes the wrong post ID to be stored.  

I verified this by deactivating all other plugins except Gutenberg, Page Builder by SiteOrigin and SiteOrigin Widgets Bundle and created a new post.  Once the post was saved as a Draft, I clicked on View Post in the Admin Bar and it took me to the wrong post.  I also verified this with the Query Monitor plugin.
